src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -183,28 +183,34 @@
 
     /**
      * The <code>options</code> file name in the output directory when calling:
-     * <code>javadoc.exe(or .sh) &#x40;options &#x40;packages | &#x40;argfile | &#x40;files</code>
+     * <code>javadoc.exe(or .sh) &#x40;options &#x40;packages | &#x40;argfile | &#x40;files | &#x40;errors</code>
      */
     protected static final String OPTIONS_FILE_NAME = "options";
 
     /**
      * The <code>packages</code> file name in the output directory when calling:
-     * <code>javadoc.exe(or .sh) &#x40;options &#x40;packages | &#x40;argfile | &#x40;files</code>
+     * <code>javadoc.exe(or .sh) &#x40;options &#x40;packages | &#x40;argfile | &#x40;files | &#x40;errors</code>
      */
     protected static final String PACKAGES_FILE_NAME = "packages";
 
     /**
      * The <code>argfile</code> file name in the output directory when calling:
-     * <code>javadoc.exe(or .sh) &#x40;options &#x40;packages | &#x40;argfile | &#x40;files</code>
+     * <code>javadoc.exe(or .sh) &#x40;options &#x40;packages | &#x40;argfile | &#x40;files | &#x40;errors</code>
      */
     protected static final String ARGFILE_FILE_NAME = "argfile";
 
     /**
      * The <code>files</code> file name in the output directory when calling:
-     * <code>javadoc.exe(or .sh) &#x40;options &#x40;packages | &#x40;argfile | &#x40;files</code>
+     * <code>javadoc.exe(or .sh) &#x40;options &#x40;packages | &#x40;argfile | &#x40;files | &#x40;errors</code>
      */
     protected static final String FILES_FILE_NAME = "files";
 
+    /**
+     * The <code>errors</code> file name in the output directory when calling:
+     * <code>javadoc.exe(or .sh) &#x40;options &#x40;packages | &#x40;argfile | &#x40;files | &#x40;errors</code>
+     */
+    protected static final String ERRORS_FILE_NAME = "errors";
```

Comment:
This does not make sense to me. Those files are passed to `javadoc(1)`. `errors` isn't passed. It is not related to the set of files.

Reply:
Just to clarify, is it the Javadoc for the constant that makes no sense or the design pattern in general? Asking to know what to change

Reply:
Those files are designed to be passed as `@arg` files. This new file, does not. It does not make logically sense to apply the same documentation.

Reply:
Got it, thanks! I went ahead and updated the Javadocs on the files